stl/inc/__msvc_int128.hpp

```diff
@@ -981,6 +994,15 @@ struct _Unsigned128 : _Base128 {
         _Word[1] |= _That._Word[1];
         return *this;
     }
+
+    _NODISCARD constexpr explicit operator bool() const noexcept {
+        return _Word[0] != 0 || _Word[1] != 0;
+    }
+
+    //for static_cast<double>
+    _NODISCARD constexpr explicit operator double() const noexcept {
+        return static_cast<double>(_Word[1]) * 18446744073709551616.0 + static_cast<double>(_Word[0]);
+    }

```

Comment:
A space should be added after `//` (ditto for below).

I think this is a valuable extension (also the STL team may reject it as a non-goal). But perhaps `operator float` & `operator long double` should be provided together to be consistent with the usual arithmetic conversions (ditto for below).

Reply:
fixed at [`2e8b843` (#3559)](https://github.com/microsoft/STL/pull/3559/commits/2e8b843c71e03f99efdb013e11964b0c1f68315d)